mgmt/tribe/worker/worker.go

```diff
@@ -545,14 +545,49 @@ func (w worker) isPluginLoaded(n, t string, v int) bool {
 }
 
 func getSchedule(s *core.Schedule) schedule.Schedule {
```

Comment:
Before:
In Tribe only Simple schedule was available

After:
- added to switch case others available schedule types
- processed here schedule validation to return only valid schedule
- added case `defaults` and log if schedule type is unrecognized